### PR TITLE
Update WMCore to py2.7/gcc493

### LINKED ISSUE #6323: Remove explicit references to python2.6
See https://www.python.org/dev/peps/pep-0394/. We can refer just to python2 which is the most up to date python 2.x series.

This pops up in just a few places:

```
-bash-4.1$ git grep -n python2.6 | grep "#"
bin/try_patch.py:1:#!/usr/bin/env python2.6
src/python/WMCore/WMRuntime/Unpacker.py:1:#!/usr/bin/env python2.6
standards/checkOutputs.py:1:#!/usr/bin/env python2.6
standards/failAnalysis.py:22:#                                                'sqlite','python2.6','standards/failAnalysis.py',
standards/failAnalysis.py:113:#  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/unittest.py", line 270, in run
standards/failAnalysis.py:130:#  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/unittest.py", line 279, in run
standards/failAnalysis.py:132:#  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/site-packages/nose-0.11.4-py2.6.egg/nose/loader.py", line 382, in loadTestsFromName
standards/failAnalysis.py:134:#  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/site-packages/nose-0.11.4-py2.6.egg/nose/importer.py", line 39, in importFromPath
standards/failAnalysis.py:136:#  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/site-packages/nose-0.11.4-py2.6.egg/nose/importer.py", line 86, in importFromDir
```

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #6323 . Since the February/2016 production deployment will be in a new arch:
- Updated all the code to the new architecture slc6_amd64_gcc493.
- no longer execute python2.6, but instead python2
- updated openssl and bz2lib path
- some examples and possibly code that we do not care anymore.
- do we need these logging?
  https://github.com/dmwm/WMCore/tree/master/test/python/WMCore_t/BossAir_t/loggingInfo

@ericvaandering @ticoann please review. Needs a throughout validation _before_ merging it.
@mmascher @hufnagel FYI

### CODE REVIEW

**Comment by mmascher** [comment]
It does not impact anything used by crab3, so green light from me.

File: bin/try_patch.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.6
+#!/usr/bin/env python2
 
 # try_patch.py -- slams the patch against jenkins
 #                   should eventually report back the status
```
Comment (ericvaandering) [inline bin/try_patch.py]: We can remove this. It was for the old version of Jenkins

File: etc/interactivejob.sh [hunk 2]
```diff
@@ -90,8 +90,8 @@ fi
 
  . $VO_CMS_SW_DIR/COMP/slc5_amd64_gcc434/external/python/2.6.4/etc/profile.d/init.sh
```
Comment (ericvaandering) [inline etc/interactivejob.sh]: What about python 2.7 here?

File: bin/try_patch.py (hunk 1 above)
Comment (amaltaro) [inline bin/try_patch.py]: remove the 2 commented lines or the file itself?

Comment (ericvaandering) [inline bin/try_patch.py]: The whole file.

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
Just tagged my branch in origin (with 1.0.12.pre3), I don't expect (and don't want) it to be pushed here, unless I made something wrong...

**Comment by amaltaro** [comment]
@ticoann I built all RPMS, deployed services in my VM and a new agent pointing to it (all with this code).
Ran all the DMWM test workflows at FNAL/CERN and didn't see any issues.

So from my side, we are good to merge this patch ;)

**Comment by ticoann** [comment]
replaced by #6502 (sorry I merged other patch and create the conflict with this)

**Pull request closed without merging**